src/libsystemd-network/sd-dhcp-client.c

```diff
@@ -41,6 +42,21 @@
 #define TRANSIENT_FAILURE_ATTEMPTS 3 /* Arbitrary limit: how many attempts are considered enough to report
                                       * transient failure. */
 
+/* RFC2131 section 4.1 specifies retransmission delays, starting at 4s then doubling
+ * each attempt with max of 64s, with -1 to +1 sec of random 'fuzz' added. */
+#define RFC2131_APPLY_FUZZ(time) \
+        usec_sub_signed((time), ((int64_t)(random_u64() % (2 * USEC_PER_SEC)) - (int64_t)USEC_PER_SEC))
+#define RFC2131_SEC41_NEXT_TIMEOUT(now, attempt) \
+        RFC2131_APPLY_FUZZ(usec_add((now), ((UINT64_C(1) << MIN((attempt) + 1, UINT64_C(6))) * USEC_PER_SEC)))
```

Comment:
Not a matter, but it is safe to drop `()` around `now`: `(now)` -> `now`

Reply:
change from macros->functions addressed this i think